### PR TITLE
iOS and Android biometrics

### PR DESCRIPTION
**Pull request by aoleszyn**
### Description
Expanded the iOS and Android biometrics.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Comment by kristofmuhi** [comment]
looks good

File: docs/mobile-apps/features/biometric-authentication.md [hunk 1]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
 * Ephemeral apps.
 :::
 
+### Android Biometrics
+
+For Android devices, there are multiple ways to configure biometrics. From API 28 and above we support the following:
+
+* [`BiometricManager`](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager) provides APIs to query if the app can authenticate and list of [Authenticators](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager.Authenticators).
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:58]:
```suggestion
* [`BiometricManager`](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager) provides APIs to query if the app can authenticate and a list of [Authenticators](https://developer.android.com/reference/android/hardware/biometrics/BiometricManager.Authenticators).
```

File: docs/mobile-apps/features/biometric-authentication.md [hunk 2]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
...
+* [`BiometricPrompt`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt) provides a user interface in the form of a dialog which is used the user to provide the finger touch and the call [`BiometricPrompt.AuthenticationCallback`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt.AuthenticationCallback) provides the response of the fingerprint Authentication.
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:59]:
```suggestion
* [`BiometricPrompt`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt) provides a user interface in the form of a dialog for the user's finger touch, and the call [`BiometricPrompt.AuthenticationCallback`](https://developer.android.com/reference/android/hardware/biometrics/BiometricPrompt.AuthenticationCallback) provides the response of the fingerprint Authentication.
```

File: docs/mobile-apps/features/biometric-authentication.md [hunk 3]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
...
+
+### iOS Biometrics
+
+For iOS devices, biometrics can be configured with different outputs. We support the following instance methods: 
+* [`evaluatePolicy`](https://developer.apple.com/documentation/localauthentication/lacontext/1514176-evaluatepolicy) evaluates the specified policy. The method [`canEvaluatePolicy`](https://developer.apple.com/documentation/localauthentication/lacontext/1514149-canevaluatepolicy) checks whether the authentication can proceed for a given policy.
+* [`biometryType`](https://developer.apple.com/documentation/localauthentication/lacontext/2867583-biometrytype) determines the type of biometric authentication method supported by the device and the method [`evaluatedPolicyDomainState`](https://developer.apple.com/documentation/localauthentication/lacontext/1514150-evaluatedpolicydomainstate) assesses the current state of the policy domain.
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:65]:
```suggestion
* [`biometryType`](https://developer.apple.com/documentation/localauthentication/lacontext/2867583-biometrytype) determines the type of biometric authentication method supported by the device, and the method [`evaluatedPolicyDomainState`](https://developer.apple.com/documentation/localauthentication/lacontext/1514150-evaluatedpolicydomainstate) assesses the current state of the policy domain.
```

File: docs/mobile-apps/features/biometric-authentication.md [hunk 4]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
...
+* [`interactionNotAllowed`](https://developer.apple.com/documentation/localauthentication/lacontext/2873508-interactionnotallowed) indicates if the authentication can be interactive.
+* [`SecItemAdd`](https://developer.apple.com/documentation/security/1401659-secitemadd) adds one or more items to a keychain. [`SecItemCopyMatching`](https://developer.apple.com/documentation/security/1398306-secitemcopymatching)returns more keychain items that match the search query. You can delete items that match the search query by using [`SecItemDelete`](https://developer.apple.com/documentation/security/1395547-secitemdelete) method. We support the following SecItem Classes: [`kSecClassGenericPassword`](https://developer.apple.com/documentation/security/ksecclassgenericpassword) and [`kSecClassInternetPassword`](https://developer.apple.com/documentation/security/ksecclassinternetpassword).
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:67]:
```suggestion
* [`SecItemAdd`](https://developer.apple.com/documentation/security/1401659-secitemadd) adds one or more items to a keychain. [`SecItemCopyMatching`](https://developer.apple.com/documentation/security/1398306-secitemcopymatching) returns more keychain items that match the search query. You can delete items that match the search query by using [`SecItemDelete`](https://developer.apple.com/documentation/security/1395547-secitemdelete) method. We support the following SecItem Classes: [`kSecClassGenericPassword`](https://developer.apple.com/documentation/security/ksecclassgenericpassword) and [`kSecClassInternetPassword`](https://developer.apple.com/documentation/security/ksecclassinternetpassword).
```

File: docs/mobile-apps/features/biometric-authentication.md [hunk 5]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
...
+
+From API 23 to 28 we support the following:
+* [`FingerprintManager.authenticate()`](https://developer.android.com/reference/android/hardware/fingerprint/FingerprintManager#authenticate(android.hardware.fingerprint.FingerprintManager.CryptoObject,%20android.os.CancellationSignal,%20int,%20android.hardware.fingerprint.FingerprintManager.AuthenticationCallback,%20android.os.Handler)) triggers the authentication.
+* [`FingerprintManager.AuthenticationCallback`](https://developer.android.com/reference/android/hardware/fingerprint/FingerprintManager.AuthenticationCallback.html) provides an instant response of the fingerprint.
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:71]: not sure what "response of the fingerprint" means here. Does this mean provides instant authentication of the fingerprint? 
```suggestion
* [`FingerprintManager.AuthenticationCallback`](https://developer.android.com/reference/android/hardware/fingerprint/FingerprintManager.AuthenticationCallback.html) provides an instant response of the fingerprint.
```

File: docs/mobile-apps/features/biometric-authentication.md [hunk 6]
```diff
@@ -49,6 +51,25 @@ Biometrics test support in Sauce Labs is not intended to test actual biometrics
...
+* [`interactionNotAllowed`](https://developer.apple.com/documentation/localauthentication/lacontext/2873508-interactionnotallowed) indicates if the authentication can be interactive.
+* [`SecItemAdd`](https://developer.apple.com/documentation/security/1401659-secitemadd) adds one or more items to a keychain. [`SecItemCopyMatching`](https://developer.apple.com/documentation/security/1398306-secitemcopymatching)returns more keychain items that match the search query. You can delete items that match the search query by using [`SecItemDelete`](https://developer.apple.com/documentation/security/1395547-secitemdelete) method. We support the following SecItem Classes: [`kSecClassGenericPassword`](https://developer.apple.com/documentation/security/ksecclassgenericpassword) and [`kSecClassInternetPassword`](https://developer.apple.com/documentation/security/ksecclassinternetpassword).
+
+From API 23 to 28 we support the following:
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/biometric-authentication.md:69]: The linked docs indicate both of these methods were deprecated in API 28.  I usually read "to" as being inclusive. : ) 

```suggestion
From API 23 to 28 we support the following:
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/mobile-apps/features/biometric-authentication.md (hunk 6 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/biometric-authentication.md:69]: @aliciakrebs  The development team confirmed that this method can be published, even if it is marked as deprecated on the android page, Sauce still supports it.

File: docs/mobile-apps/features/biometric-authentication.md (hunk 5 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/biometric-authentication.md:71]: This call is responsible for providing a response, not instant authentication. So the response could be in a form of error messages, successful/unsuccessful authentication etc.

**Pull request merged**